.github/workflows/internal-release.yml

```
@@ -66,8 +66,8 @@ jobs:
 
       - name: Set output
         id: set_output
-        run: echo "::set-output name=VERSION_NAME::${{ env.VERSION_NAME }}" && echo "::set-output name=VERSION_CODE::${{ env.VERSION_CODE }}"
-
+        run: echo "VERSION_CODE=${{ env.VERSION_CODE }}" >> $GITHUB_OUTPUT
+        run: echo "VERSION_NAME=${{ env.VERSION_NAME }}" >> $GITHUB_OUTPUT
```

```suggestion
        run: echo "VERSION_CODE=${{ env.VERSION_CODE }}" >> $GITHUB_OUTPUT && echo "VERSION_NAME=${{ env.VERSION_NAME }}" >> $GITHUB_OUTPUT
```

Not familiar with the `.yml` syntax but it looks like we can have only one `run`.
Therefore, the shell `&&` seems appropriate to execute a list of commands.